app/assets/javascripts/discourse/app/components/sidebar/api-sections.gjs

```diff
@@ -82,6 +88,46 @@ function prepareSidebarSectionClass(Section) {
       });
     }
 
+    get activeLink() {
+      return this.filteredLinks.find((link) => {
+        try {
+          const currentWhen = link.currentWhen;
+
+          if (typeof currentWhen === "boolean") {
+            return currentWhen;
+          }
+
+          // TODO detect active links using the href field
+
+          const queryParams = link.query || {};
+          let models;
+
+          if (link.model) {
+            models = [link.model];
+          } else if (link.models) {
+            models = link.models;
+          } else {
+            models = [];
+          }
```

Comment:
```suggestion
const models = [link.model || link.models]
```

Reply:
Unfortunately, this doesn't work, as it would create a nested array for models.

`[link.model || ...link.models]` would fail if models is null.

Reply:
Ahh you are right 👏